src/js/util/AuthService.jsx

```diff
@@ -11,8 +11,10 @@ var url = "https://dev.calligre.com"
 class AuthService extends EventEmitter {
   constructor(clientId, domain) {
     super()
+    this.clientId = clientId
+    this.domain = domain
     // Configure Auth0
-    this.lock = new Auth0Lock(clientId, domain, {auth: {redirect: false}})
+    this.lock = new Auth0Lock(clientId, domain, {auth: {redirect: true}})
```

Comment:
Also, I'm still running into the problem where redirect true is not working. I either redirects to the login page or goes to an invalid url.